src/System.Configuration.ConfigurationManager/src/System/Configuration/ConfigurationPermission.cs

```diff
@@ -0,0 +1,195 @@
+//------------------------------------------------------------------------------
+// <copyright file="ConfigurationPermission.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+
+namespace System.Configuration {
+
+    using System.Security;
+    using System.Security.Permissions;
+    using System.Globalization;
+    using System.Diagnostics.CodeAnalysis;
+    
+    [AttributeUsage(AttributeTargets.All, AllowMultiple=true, Inherited=false )]
+    [Serializable] 
+    sealed public class ConfigurationPermissionAttribute : CodeAccessSecurityAttribute
```

Comment:
We don't need any of this. All the Permission attributes should be basically dummies. Take a look eg at src\System.Security.Permissions\src\System\Net\Mail\SmtpPermission.cs. They should not throw but not do anything either.